docker/images/n8n-custom/Dockerfile

```diff
@@ -11,14 +11,14 @@ RUN apk --update add --virtual build-dependencies python3 build-base ca-certific
 WORKDIR /data
 
 COPY lerna.json .
-COPY package.json .
-COPY package-lock.json .
-COPY packages/cli/ ./packages/cli/
-COPY packages/core/ ./packages/core/
-COPY packages/design-system/ ./packages/design-system/
-COPY packages/editor-ui/ ./packages/editor-ui/
-COPY packages/nodes-base/ ./packages/nodes-base/
-COPY packages/workflow/ ./packages/workflow/
+COPY package.json package*.json .
+COPY packages/* ./packages
+#COPY packages/cli/ ./packages/cli/
+#COPY packages/core/ ./packages/core/
+#COPY packages/design-system/ ./packages/design-system/
+#COPY packages/editor-ui/ ./packages/editor-ui/
+#COPY packages/nodes-base/ ./packages/nodes-base/
+#COPY packages/workflow/ ./packages/workflow/
```

Comment:
This is now including `packages/node-dev`

Reply:
please merge this request with https://github.com/n8n-io/n8n/pull/3846